src/@types/emitter-listener.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module 'emitter-listener' {
```

Comment:
Could you submit this to DefinitelyTyped instead?

Reply:
Yes, but in the meantime it should not block this PR.